HoloPipelines/models/dense_vnet_abdominal_ct/server.py

```diff
@@ -55,4 +58,4 @@ def seg_file():
 
 
 if __name__ == "__main__":
-    app.run(debug=False, host="127.0.0.1")
+    app.run(debug=True, host="127.0.0.1")
```

Comment:
👎

Reply:
You can make it debug mode conditionally; take a look how it is handled in the main pipelines server.